### PR TITLE
byte[] support in service response decoder

### PR DESCRIPTION
**Pull request by psrkprasad1995**

### CODE REVIEW

File: service-clients-core/src/main/java/com/flipkart/poseidon/serviceclients/ServiceResponseDecoder.java [hunk 1]
```diff
@@ -96,6 +96,9 @@ public ServiceResponseDecoder(ObjectMapper objectMapper, Logger logger, Map<Stri
                 try {
                     // Don't deserialize a plain string response using jackson
                     final JavaType javaType = serviceResponseInfoMap.get("200").getType();
+                    if (byte[].class.isAssignableFrom(javaType.getRawClass())) {
```
Comment (shrey-garg) [inline service-clients-core/src/main/java/com/flipkart/poseidon/serviceclients/ServiceResponseDecoder.java]: put this as an else after the String check

File: api/pom.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <parent>
         <groupId>com.flipkart.poseidon</groupId>
         <artifactId>poseidon</artifactId>
-        <version>5.6.0-SNAPSHOT</version>
+        <version>5.6.0-SRK-SNAPSHOT</version>
```
Comment (shrey-garg) [inline api/pom.xml]: revert these

**Comment by pmohankumar** [comment]
@psrkprasad1995 , please fix the review comments.

**Comment by pmohankumar** [comment]
@psrkprasad1995 , can you revert pom file (version) changes as suggested by Shrey.

**Pull request merged**